krun/platform.py

```diff
@@ -513,6 +551,12 @@ def is_virtual(self):
 
         return False
 
+    def get_allowed_dmesg_patterns(self):
+        # PyPy uses write+executable pages, which causes this dmesg entry
+        # in OpenBSD-current (as of around 6.0-beta)
+        return UnixLikePlatform.get_allowed_dmesg_patterns(self) + \
+            [re.compile("pypy\([0-9]+\): mmap W\^X violation$")]
```

Comment:
Should we be hard-coding this into Krun, or allowing the user to specify it in a config file?